packages/introspect/src/tests/sqlQueries.ts

```diff
@@ -0,0 +1,159 @@
+export const createTableQuery = `
```

Comment:
Did you consider generating these queries dynamically, to avoid all the repetitive noise:

In essence you need:

1. A type (e.g. SERIAL PRIMARY KEY / BIGINT / BOOLEAN)
2. An example, so you can write it to the table (e. . 1 / 1 / true)

With this we can create something like:

```
const allKnownTypes:Array<Thing> = [
    {type: 'SERIAL PRIMARY KEY', examples:['1','9223372036854775807']},
    {type: 'BOOLEAN', examples: ['true,false']},
    ....
]

function getColumnName(x:Thing) {
    return `col_${normalize(x.type)}`
}
```

You can then generate the select and insert pretty easily:

```
CREATE TABLE all_data_types (
    ${allKnownTypes.map(x => ...)}.join(",")
)
```